endpoints/cookie_sync_test.go

```diff
@@ -1581,18 +1615,29 @@ func TestCookieSyncHandleResponse(t *testing.T) {
 	syncerWithError := MockSyncer{}
 	syncerWithError.On("GetSync", syncTypeExpected, privacyMacros).Return(syncWithError, errors.New("anyError")).Maybe()
 
+	bidderEvalForDebug := []usersync.BidderEvaluation{
+		{Bidder: "Bidder1", Status: usersync.StatusAlreadySynced},
+		{Bidder: "Bidder2", Status: usersync.StatusUnknownBidder},
+		{Bidder: "Bidder3", Status: usersync.StatusUnconfiguredBidder},
+		{Bidder: "Bidder4", Status: usersync.StatusBlockedByPrivacy},
+		{Bidder: "Bidder5", Status: usersync.StatusTypeNotSupported},
+		{Bidder: "DuplicateBidder", Status: usersync.StatusDuplicate, SyncerKey: "syncer"},
+		{Bidder: "DuplicateBidder", Status: usersync.StatusDuplicate, SyncerKey: "syncer"},
+	}
```

Comment:
Please also add a test case of `StatusDuplicate` where the same bidder name is in the list twice. This is possible due to how the list of prospective bidders are constructed and should be filtered out / ignored.